packages/terra-tabs/tests/nightwatch/Tabs/ControlledTabs.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import Base from 'terra-base';
-import Tabs from '../../../lib/Tabs';
+import Tabs from '../../../src/Tabs';
```

Comment:
We should probably switch this back to `lib`

Reply:
I was debugging Tabs and didn't realize the change got committed. Reverted. https://github.com/cerner/terra-core/pull/1155/commits/81a22fc16181f57f506752dc4df5abb6c6eeea55